SPECS/openssh.spec

```diff
@@ -471,9 +477,17 @@ popd
 
 %check
 #to run tests use "--with check"
-%if %{?_with_check:1}%{!?_with_check:0}
-make tests
-%endif
+#%if %{?_with_check:1}%{!?_with_check:0}
+if ! command -v sudo >/dev/null 2>&1; then
+	echo "command sudo not available, test will be ignored."
+else
```

Comment:
A comment mentioning in which case this is useful (koji?) would be nice.
Maybe add a "WARNING:" prefix to the message so it is more visible?